docs/support.md

```diff
@@ -22,6 +22,6 @@ Many developers and users idle on Freenode.net's IRC network in **[#reactjs on f
 
 ## Twitter
 
-For the latest news about React, [follow **@reactjs** on Twitter](https://twitter.com/reactjs). In addition, you can use the [**#reactjs** hashtag](https://twitter.com/search?q=%23reactjs) to keep up with the latest React news.
+For the latest news about React, [follow **@reactjs** on Twitter](https://twitter.com/reactjs) or use the [**#reactjs** hashtag](https://twitter.com/search?q=%23reactjs).
```

Comment:
Thanks Jim. I definitely see your point there. I can edit the change to be 'and' or 'and/or' depending on what you guys prefer, since if for whatever reason a twitter user does not wish to follow reactjs but still wishes to check the latest news, they can use the hashtag.